recipes/cortecs/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "cortecs" %}
+{% set version = "0.2.0b0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cortecs-{{ version }}.tar.gz
+  sha256: b498807e17a93fdea2cc07fc25a10bfc3a490b3f364ef11db336d8e49ec2a39b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python >=3.9,<3.12
+    - pip
+  run:
+    - python >=3.9,<3.12
+    - numpy
+    - conda-forge::jax >=0.4.20,<0.5.0
```

You don't need this to build in conda-forge's infrastucture.

```suggestion
    - jax >=0.4.20,<0.5.0
```